src/app/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Router, Switch, Route } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
+
+import {
+  PopularPage,
+  UpcomingPage,
+  NowPlayingPage,
+} from '../pages';
+import Header from '../components/Header';
+
+import './app.css';
+
+const history = createBrowserHistory();
+
+function App() {
+  return (
+    <div className="wrapper">
+      <div className="app">
+        <Router history={history}>
+          <Header history={history} />
+          <Switch>
+            <Route exact path="/" component={PopularPage} />
```

Comment:
Try to use constants for paths. For example, instead '/' use ROUTES.POPULAR, instead '/upcoming' - ROUTES.UPCOMING and so on.